app/src/androidTest/java/com/percy/espresso_java/AppPercyTest.java

```diff
@@ -35,14 +34,12 @@ public class AppPercyTest {
     WireMockServer server = new WireMockServer(5338);
     GenericProvider genericProvider = new GenericProvider();
 
-    Environment environment = new Environment();
-
     @Before
     public void setup() throws JSONException {
         CliWrapper.PERCY_SERVER_ADDRESS = "http://127.0.0.1:5338";
         genericProvider.setMetadata(new Metadata(new ScreenshotOptions()));
         ResponseDefinitionBuilder mockResponse = new ResponseDefinitionBuilder();
-        mockResponse.withHeader("x-percy-core-version", "1.2");
+        mockResponse.withHeader("x-percy-core-version", "1.23.0");
```

Comment:
have we tested anything here. I assume that this test will not fail even if we remove this.
We can add test for log message if we are seeing it or not.

Reply:
[Optional] Can we add test if we are seeing that log printed here.

Reply:
jest test doesn’t have a good way for checking logline print, its mainly checks for fn return value